lib/server/websocket.js

```diff
@@ -405,20 +406,22 @@ function init (env, ctx, server) {
               }
               // if not found create new record
               console.log(LOG_DEDUP + 'Adding new record');
-              ctx.store.collection(collection).insert(data.data, function insertResult (err, doc) {
+              ctx.store.collection(collection).insertOne(data.data, function insertResult (err, ops) {
```

Comment:
AAPS folks: do multiple documents come through this interface as an array?  Can you provide any examples?